Gir.toml

```diff
@@ -612,6 +612,45 @@ status = "generate"
     ignore = true
 
 [[object]]
+name = "Gtk.TreeModelFilter"
+status = "generate"
+    [[object.function]]
+    name = "clear_cache"
+    [[object.function]]
+    name = "get_model"
```

Comment:
It seems this widget should always have an associated model so we could make this function non-nullable.